frontend/app/src/components/trades/ClosedTrades.vue

```diff
@@ -46,6 +49,8 @@
             <template #item.fee="{ item }">
               <amount-display
                 class="closed-trades__trade__fee"
+                show
```

Comment:
Is this a new flag?

Reply:
nope, I don't remember adding new flags there, it seems like an accidental addition.